From be85905e77ee4b20504b395b4e67b179f545d642 Mon Sep 17 00:00:00 2001
From: Reto Schneider <reto.schneider@husqvarnagroup.com>
Date: Thu, 4 Mar 2021 02:51:41 +0100
Subject: [PATCH] shared library: Link against zlib

Library is using crc32(), therefore needs to link against zlib.
---
 src/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index a04dd11..bd38366 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -19,6 +19,7 @@ ADD_LIBRARY(ubootenv_static STATIC ${libubootenv_SOURCES} ${include_HEADERS})
 SET_TARGET_PROPERTIES(ubootenv_static PROPERTIES OUTPUT_NAME ubootenv)
 add_executable(fw_printenv fw_printenv.c)
 add_executable(fw_setenv fw_setenv.c)
+target_link_libraries(ubootenv z)
 target_link_libraries(fw_printenv ubootenv z)
 target_link_libraries(fw_setenv ubootenv z)
 
-- 
2.29.2

